CHANGELOG.rst

```diff
@@ -1,7 +1,8 @@
-Jinja Filters Changelog
-===============
+CHANGELOG
+=========
 
-This Changelog is for ``minchin.pelican.jinja_filters``.
+1.0.4 - 2017-04-17
```

Comment:
Is there a reason to drop the reference to the project title here?

FYI the format is for [releases](https://github.com/bitprophet/releases) for Sphinx.

As well, I'll need to add an entry for this new version once this is merged.

Reply:
[AutoPub](https://github.com/autopub/autopub) looks for certain patterns when it automatically updates the changelog, and I don't think there's currently any way for it to preserve this header — it would most likely get pushed down below, with the next version's info prepended above it. That's why I removed the header. You can see an example of the "standard" changelog format here: https://raw.githubusercontent.com/pelican-plugins/simple-footnotes/master/CHANGELOG.md

I'm sure [AutoPub](https://github.com/autopub/autopub) could be made more flexible is this regard, but so far the need hasn't come up. At some point, perhaps we could add more flexibility to [AutoPub](https://github.com/autopub/autopub) to support these kinds of persistent headers/titles.